doc/Language/signatures.rakudoc

```diff
@@ -986,6 +986,59 @@ $sig1 = :($goal, $accuracy = $goal / 100);
 $sig2 = :(:$excludes = ['.', '..']);        # a new Array for every call
 =end code
 
+U<Was an argument passed for a parameter?>
```

Comment:
Please change this to a =head2 - then it will get an appropriate anchor and show up in the TOC.

Reply:
```suggestion
=head2 Was an argument passed for a parameter?
```